Formula/stubby.rb

```diff
@@ -0,0 +1,93 @@
+class Stubby < Formula
+  desc "DNS privacy enabled stub resolver based on getdns"
+  homepage "https://getdnsapi.net"
+  url "https://getdnsapi.net/releases/getdns-1-1-1rc1/getdns-1.1.1rc1.tar.gz"
+  sha256 "f63340b1d05410b875217c6abd7066586fc55a811db4ae90ffd01d2240e05e57"
+  head "https://github.com/getdnsapi/getdns.git", :branch => "develop"
+
+  depends_on "openssl"
+  depends_on "libevent" => :recommended
+  depends_on "libuv" => :optional
+  depends_on "libev" => :optional
+
+  if build.head?
+    depends_on "libtool"
+    depends_on "autoconf"
+    depends_on "automake"
+  end
+
+  def install
+    if build.head?
+      system "glibtoolize", "-ci"
+      system "autoreconf", "-fi"
+    end
+
+    args = [
+      "--with-ssl=#{Formula["openssl"].opt_prefix}",
+      "--with-trust-anchor=#{etc}/getdns-root.key",
+    ]
+    args << "--with-libevent" if build.with? "libevent"
+    args << "--with-libuv" if build.with? "libuv"
+    args << "--with-libev" if build.with? "libev"
+
+    # Current Makefile layout prevents simultaneous job execution
+    # https://github.com/getdnsapi/getdns/issues/166
```

Comment:
The issue is closed but when I tried removing this when building on my machine it caused the build to fail. I am talking to the other developers about this but wanted to leave it in for now to be on the safe side.